core/src/main/scala/zio/oci/objectstorage/options.scala

```diff
@@ -25,6 +25,14 @@ object ListObjectsOptions {
   def oneAfter(name: String): ListObjectsOptions = ListObjectsOptions(None, None, Some(name), 1, Set(Field.Name, Field.Size))
 }
 
+final case class GetObjectOptions(range: Option[GetObjectOptions.Range])
+
+object GetObjectOptions {
+  final case class Range(startByte: Option[Long], endByte: Option[Long])
```

Comment:
As is there’s a fourth case (None, None) that we probably need to see how it would be handled by the OCI api. Alternatively we could make that case unrepresentable, wdyt?

Reply:
we certainly can but it won't be a problem for oci sdk as values will be set to null in case of None.